src/jetstream.rs

```diff
@@ -57,7 +57,7 @@
 //!
 //! js.add_stream("my_stream")?;
 //!
-//! let consumer: nats::jetstream::Consumer = js.add_consumer("my_stream", "my_consumer")?;
+//! let consumer: nats::jetstream::ConsumerSubscription = js.add_consumer("my_stream", "my_consumer")?;
```

Comment:
Consumers are separate from Subscriptions. This might need to be reconsidered a bit.

Reply:
I think Consumer is more correct here.